src/app/rating/rating-list/rating-list.component.html

```diff
@@ -0,0 +1,33 @@
+<app-rating-search [pageSize] = "pageSize" (emitResults)="modifyList($event)"></app-rating-search>
+<div class="card mb-1" *ngFor="let rating of ratings">
+  <div class="card-block row">
+    <div class="card-body row align-items-center">
+      <div class="col">
+        <h6 class="card-subtitle text-muted">Comment</h6>
+        <a class="card-text" [routerLink]="['/ratings', rating.id]">{{rating.comment}}</a>
```

Comment:
With the current API, the `id` is not returned in the JSON of any object. You need to expose the `id` of the different classes in the way your classmates are doing in the file `ExposeIdConfig` on the API. This is causing this links to not work.